bigquery_etl/query_scheduling/task.py

```diff
@@ -280,6 +281,25 @@ def with_dependencies(self, client, dag_collection):
                     and d.task_id == upstream_task.task_name
                     for d in self.depends_on
                 ):
-                    dependencies.append(upstream_task)
+                    upstream_schedule_interval = dag_collection.dag_by_name(
```

Comment:
For the dependencies that are automatically determined we have to calculate the timedelta between the scheduling intervals of the different DAGs. This is done here